packages/suite-desktop/src-electron/libs/processes/BaseProcess.ts

```diff
@@ -48,6 +50,15 @@ abstract class BaseProcess {
             ...defaultOptions,
             ...options,
         };
+
+        const { logger } = global;
+        this.logger = logger;
+        this.logTopic = `process-${this.processName}`;
+
+        const { system } = this.getPlatformInfo();
+        if (!this.isSystemSupported(system)) {
+            throw new Error(`[${this.resourceName}] unsupported system (${system})`);
```

Comment:
shouldn't this be in the logger?
throw error in process constructor may cause app crashing, no? 
or is it handled in try/catch somewhere?

Reply:
The logger will catch unhandled events but I think we might want to limit the usage of Suite to supported systems somewhere else maybe.

Reply:
Addressed in  417a0f7. Changed it to log it as an error.